packages/ng/libraries/root/src/style/definitions/select/_select-input.scss

```diff
@@ -133,9 +134,18 @@
 			}
 
 			&.mod-multiple.mod-multipleView.is-filled .lu-select-value {
-				margin-bottom: -.3rem;
+				padding-bottom: _component("select.framed.bottom-padding", true);
+			}
+			&.mod-multiple.mod-multipleView:not(.mod-singleLine) .lu-select-value {
+					min-height: _component("select.line-height");
+			}
+			&.mod-multiple{
+				padding: calc(#{_component("select.framed.top-padding", true)} - .2rem ) 0 0;
 			}
 		}
+		& .mod-small.textfield-input {
+			min-height: 3.875rem;
```

Comment:
Why this value ? it seems oddly specific. .875rem is the font-size "small". I guess the 3rem comes from somewhere else. It could be good to know as it will make it resilient to changes

Reply:
This was a test, but I forgot remove. I'll do it right away.